instant-markdown-d

```diff
@@ -1,10 +1,10 @@
-#!/bin/sh
-':' //; exec "`command -v nodejs || command -v node`" "$0"
+#!/usr/bin/env node
```

Comment:
Ah. Actually it seems the solution is to have any Ubuntu/Debian users install the nodejs-legacy package, which provides the symlink I suggested.